core/spec.md

```diff
@@ -247,7 +247,7 @@ adhere to the following rules:
   treated the same as an attribute with no value.
 - the string serialization of the attribute name and its value MUST NOT exceed
   4096 bytes. This is to ensure that it can appear in an HTTP header without
-  exceeding implemantion limits (see
+  exceeding implemention limits (see
```

Comment:
Maybe you'll get lucky third time? ;)

Reply:
I find my typing is getting worse as I get older ;-)  
Thanks for the review